.github/workflows/build-example-web.yml

```
@@ -8,8 +8,14 @@ jobs:
     env:
       TEST_ENDPOINT: hardware-example.onekeytest.com
       PROJECT_PATH: packages/connect-examples/expo-example
+      COMMIT_SHA: ${{ github.sha }}
+      COMMIT_SHORT_SHA: ${{ github.sha }}
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)